src/npc.cpp

```diff
@@ -255,7 +264,16 @@ void Npc::onCreatureAppear(Creature* creature, bool isLogin)
 	Creature::onCreatureAppear(creature, isLogin);
 
 	if (creature == this) {
-		if (walkTicks > 0) {
+		SpectatorVec players;
```

Comment:
the redundancy of the word `player` `players` `player`

Reply:
What?

Reply:
Don't worry, I was thinking of changing the name of the vector, but I already checked it well and there is nothing wrong, excuse me